apiary.apib

```diff
@@ -652,14 +652,11 @@ A list of objects containing the following parameters:
 
 ### Add an item to the Queue [POST]
 
-A new item can be put in the Queue, this item will auto expire after 30 seconds. In order to make sure that your item stays alive you will need to repeat the request with the same ID.
```

Comment:
That statement about 30 seconds, it is still true. I think it is better to have it in the documentation